packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -835,75 +844,110 @@ class JavaGenerator extends Generator {
         }
 
         collectProps(ifc);
+        const props = Array.from(propsByName.values());
 
-        this.code.line();
+        // Start Builder
         this.code.line('/**');
         this.code.line(` * A builder for {@link ${interfaceName}}`);
         this.code.line(' */');
         this.emitStabilityAnnotations(ifc);
         this.code.openBlock(`final class ${builderName}`);
-
-        for (const prop of props) {
-            if (prop.nullable) {
-                this.code.line(JSR305_NULLABLE);
-            }
-            this.code.line(`private ${prop.fieldJavaType} _${prop.fieldName};`);
-        }
+        props.forEach(prop => this.code.line(`private ${prop.fieldJavaType} ${prop.fieldName};`));
```

Comment:
I tend to prefer the `for (... of ...)` syntax as much as possible. The callback form actually makes TypeScript's work harder, and pre-existing type-constraining information is lost when we get into the callback... I don't think it makes a difference for this particular case, but still... consistency :)